packages/modules/postgresql/src/postgresql-container.ts

```diff
@@ -85,4 +86,100 @@ export class StartedPostgreSqlContainer extends AbstractStartedContainer {
     url.password = this.getPassword();
     return url.toString();
   }
+
+  public withSnapshotName(snapshotName: string): this {
+    this.snapshotName = snapshotName;
+    return this;
+  }
+
+  /**
+   * Takes a snapshot of the current state of the database as a template, which can then be restored using
+   * the restore method. By default, the snapshot will be created under a database called migrated_template.
+   *
+   * If a snapshot already exists under the given/default name, it will be overwritten with the new snapshot.
+   *
+   * @param opts Optional snapshot configuration options
+   * @returns Promise resolving when snapshot is complete
+   */
+  public async snapshot(snapshotName?: string): Promise<void> {
+    const name = snapshotName || this.snapshotName;
```

Comment:
```suggestion
    const name = snapshotName ?? this.snapshotName;
```

Reply:
Yeah, it's a bit cleaner with the default arg, updated